Documentation/faq.md

```diff
@@ -1,5 +1,11 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, general
+
+#### Does the clients need to send requests to the leader?
```

Comment:
i ask those question because i don't really know how client really work. i suppose future  etcd user might have the same questions.